src/main/java/com/synopsys/integration/detect/lifecycle/boot/decision/BlackDuckDecision.java

```diff
@@ -6,23 +6,26 @@ public class BlackDuckDecision {
     private final boolean shouldRun;
     private final boolean isOffline;
     private final BlackduckScanMode scanMode;
+    private final boolean hasSignatureScanner;
 
-    public BlackDuckDecision(boolean shouldRun, boolean isOffline, BlackduckScanMode scanMode) {
+    public BlackDuckDecision(boolean shouldRun, boolean isOffline, BlackduckScanMode scanMode, boolean hasSigScan) {
         this.shouldRun = shouldRun;
         this.isOffline = isOffline;
         this.scanMode = scanMode;
+        this.hasSignatureScanner = (hasSigScan) ? true : false;
```

Comment:
The conditional operator doesn't add any value here, does it? Why not just assign hasSigScan to it?

Reply:
Not sure but that's likely a holdover from a prior attempt.  You're right that this isn't too useful